configs/ipq6018_defconfig

```diff
@@ -1,320 +1,28 @@
 CONFIG_ARM=y
-CONFIG_HAS_VBAR=y
-CONFIG_CPU_V7=y
 CONFIG_ARCH_IPQ6018=y
-CONFIG_SYS_MALLOC_F_LEN=0x400
-CONFIG_SYS_MALLOC_F=y
-CONFIG_DM_SERIAL=y
-CONFIG_DEFAULT_DEVICE_TREE=""
-CONFIG_LOCALVERSION=""
-CONFIG_LOCALVERSION_AUTO=y
-CONFIG_CC_OPTIMIZE_FOR_SIZE=y
-CONFIG_EXPERT=y
-CONFIG_SYS_MALLOC_CLEAR_ON_INIT=y
+CONFIG_USB_XHCI_IPQ=y
+CONFIG_PCI_IPQ=y
+CONFIG_IPQ6018_QCA_AQUANTIA_PHY=y
+CONFIG_QCA8081_PHY=y
+CONFIG_QCA8075_PHY=y
+CONFIG_DEFAULT_DEVICE_TREE="ipq6018-siklu-ctu-100"
```

Comment:
This is not the name of the cTU device-tree file.

This update of `ipq6018_defconfig` adds lots of noise to this commit for now apparent reason. Please drop these changes.